### PR TITLE
Adding isHelmV3 check to avoid breaking changes

### PR DESCRIPTION
**Pull request by shigupt202**

### CODE REVIEW

File: Tasks/HelmDeployV0/src/helmcli.ts [hunk 1]
```diff
@@ -53,8 +54,9 @@ export default class helmcli extends basecommand {
     }
 
     public isHelmV3(): boolean {
-        var helmVersion = this.getHelmVersion();
-        if(helmVersion.stdout.charAt(1) == '3')
+        if(!this.helmVersion)
+            this.helmVersion = this.getHelmVersion().stdout;
+        if(this.helmVersion.charAt(1) == '3')
```
Comment (ds-ms) [inline Tasks/HelmDeployV0/src/helmcli.ts:59]: Use starts with `v3` instead?

Comment (ds-ms) [reply Tasks/HelmDeployV0/src/helmcli.ts:59]: Fix indent

File: Tasks/HelmDeployV0/src/helmcommands/helmpackage.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export function addArguments(helmCli: helmcli) : void {
         helmCli.addArgument("--dependency-update");
     }
 
-    if(save) {
+    if(save && !helmCli.isHelmV3()) {
```
Comment (ds-ms) [inline Tasks/HelmDeployV0/src/helmcommands/helmpackage.ts:20]: Fix indent here as well

Comment (ds-ms) [reply Tasks/HelmDeployV0/src/helmcommands/helmpackage.ts:20]: Does it save by default for `v2`?

Comment (shigupt202) [reply Tasks/HelmDeployV0/src/helmcommands/helmpackage.ts:20]: Yes. The default value is true for `save`

**Pull request merged**